Katerina Shemet/Lesson7.Homework/Program.cs

```diff
@@ -0,0 +1,142 @@
+﻿using System;
+
+class Program
+{
+    static void sortStrings(String[] arr, int n)
+    {
+        String temp;
+
+        for (int j = 0; j < n - 1; j++)
+        {
+            for (int i = j + 1; i < n; i++)
+            {
+                if (arr[j].CompareTo(arr[i]) > 0)
+                {
+                    temp = arr[j];
+                    arr[j] = arr[i];
+                    arr[i] = temp;
+                }
+            }
+        }
+    }
+
+static bool stringCompare(String strng1, String strng2)
+{
+    char[] charArray1 = strng1.ToCharArray();
+    char[] charArray2 = strng2.ToCharArray();
+
+    if(charArray1.Length != charArray2.Length)
+    {
+        return false;
+    }
+
+    for (int i = 0; i < charArray1.Length; i++)
+    {
+            if (charArray1[i]!=charArray2[i])
+            {
+                return false;
+            }
+    }
+        return true;
+}
+
+    static void stringAnalyze(String str)
+    {
+        int alp, digit, splch, i, l;
+        alp = digit = splch = i = 0;
+        
+        l = str.Length;
+
+        while (i < l)
+        {
+            if ((str[i] >= 'a' && str[i] <= 'z') || (str[i] >= 'A' && str[i] <= 'Z'))
```

Comment:
char class have build in methods to check if it number or letter, we have covered it in classwork